spark/src/main/java/org/opensearch/sql/spark/storage/SparkScan.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.storage;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.opensearch.sql.data.model.ExprValue;
+import org.opensearch.sql.spark.client.SparkClient;
+import org.opensearch.sql.spark.request.SparkQueryRequest;
+import org.opensearch.sql.storage.TableScanOperator;
+
+/**
+ * Spark scan operator.
+ */
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
+@ToString(onlyExplicitlyIncluded = true)
+public class SparkScan extends TableScanOperator {
```

Comment:
What's the difference between this class and SparkSqlFunctionTableScanOperator?

Reply:
this class returns false always. will address this  separately in refactor https://github.com/opensearch-project/sql/issues/1820